test/matchers/shared_examples/has_valid_rspec_message_methods.rb

```diff
@@ -59,6 +64,7 @@ module HasValidRspecMessageMethods
         case validator_sym
         when :aspect_ratio then matcher.allowing(:square)
         when :attached then matcher
+        when :limit then matcher
```

Comment:
That's the inverse of the previous test, it should make the matcher return `true`, so it could be `when :limit then matcher.max(5)` for example